src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -201,8 +206,13 @@ export default {
   }
 }
 
+.rounded {
```

Comment:
Please use component namespacing as we don't have scoped styles at the moment. `oc-drop-rounded`.

Reply:
alright thanks for the hint will try to remember next time ^^